mpsl/include/mpsl_clock.h

```diff
@@ -103,6 +103,11 @@ typedef struct
    *  If it is set to true, MPSL will wait for the low frequency clock later,
    *  before the low frequency clock is used for the first time.
    *  Default value is @ref MPSL_DEFAULT_SKIP_WAIT_LFCLK_STARTED.
+   *
+   * @note If this option is set to true and the application starts Low Frequency clock,
+   *       the application shall either stop Low Frequency clock
+   *       or not clear the event signaling that Low Frequency clock has started
+   *       before MPSL initialization.
```

Comment:
What about 
```suggestion
   * @note If this option is set to true and the application writes to
   *       NRF_CLOCK->TASKS_LFCLKSTART before mpsl_init() is called,
   *       the application shall either:
   *       - Write to NRF_CLOCK->TASKS_LFCLKSTOP.
   *       - Keep NRF_CLOCK->EVENTS_LFCLKSTARTED untouched.
```

Reply:
Sure

Reply:
Is this leftover from SoftDevice behavior? This could be handled by the LFCLK driver, unless the application writes it's own driver, right?

Reply:
Done

Reply:
Need to make sure 378 would revert this change.